src/amo/components/CollectionAddAddon/styles.scss

```diff
@@ -0,0 +1,35 @@
+@import '~photon-colors/photon-colors';
+@import '~amo/css/inc/vars';
+
+.CollectionAddAddon {
+  .ErrorList,
+  .NoticePlaceholder {
+    margin-top: 16px;
+    min-height: 32px;
+  }
+
+  .AutoSearchInput-label {
+    padding-top: 4px;
+  }
+
+  .AutoSearchInput-query {
+    border: 1px solid $grey-50;
+    border-radius: $border-radius-xs;
+  }
+
+  .AutoSearchInput--autocompleteIsOpen .AutoSearchInput-query {
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+  }
+
+  .Notice {
```

Comment:
This is a bit confusing because I don't see any elements getting rendered underneath `<Notice />`. Could you add a comment about why it's needed? Maybe it's because the css transition component alters the dom?

As an aside, this transition functionality seems like it should live in `Notice` but we could worry about that later, when we want to add transitions to another notice.